views/templates/admin/tabs/help.tpl

```diff
@@ -0,0 +1,94 @@
+{*
+* 2007-2018 PrestaShop
+*
+* DISCLAIMER
+*
+* Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+* versions in the future. If you wish to customize PrestaShop for your
+* needs please refer to http://www.prestashop.com for more information.
+*
+* @author    PrestaShop SA <contact@prestashop.com>
+* @copyright 2007-2018 PrestaShop SA
+* @license   http://addons.prestashop.com/en/content/12-terms-and-conditions-of-use
+* International Registered Trademark & Property of PrestaShop SA
+*}
+
+<div class="panel col-lg-10 right-panel">
+    <h3>
+        <i class="fa fa-question-circle"></i> {l s='Help & contact' d='Modules.Faviconnotificationbo.Admin'} <small>{$module_display|escape:'htmlall':'UTF-8'}</small>
+    </h3>
+    <div class="helpContentParent">
+        <div class="helpContentLeft">
+            <div class="left">
+                <img src="{$logo_path|escape:'htmlall':'UTF-8'}" alt=""/>
+            </div>
+            <div class="right">
+                <p><span class="data_label" style="color:#00aff0;"><b>{l s='This module allows you to :' d='Modules.Faviconnotificationbo.Admin'}</b></span></p>
```

Comment:
Watch the space before the colon. English grammar removes it. It should be _This module allows you to:_ then. Perhaps in the _Admin.Global_ since it is a generic wording that can be used elsewhere in the back office.